ex_5_v2.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jun  5 15:36:53 2018
+
+@author: amyhaddad
+"""
+
+number_1 = input("What is the first number? ")
+number_2 = input("What is the second number? ")
```

Comment:
Perfect here!

Reply:
Everything looks great here. To reduce the repetition of converting the input to integers down below, you can do something like this:

```
number_1 = int(input("What is the first number? "))
number_2 = int(input("What is the second number? "))
```

Then you can simply do `sum = number_1 + number_2`, etc, which simplifies everything and makes it easier to read.

Reply:
Agreed. I made the change!